### PR TITLE
JAVA-3219 : Update spring-security registration libraries

### PR DESCRIPTION
**Pull request by anuragkumawat**

### CODE REVIEW

File: src/main/java/com/baeldung/spring/MvcConfig.java [hunk 1]
```diff
@@ -107,4 +109,8 @@ public Validator getValidator() {
         return validator;
     }
 
+    @Bean
+    WebServerFactoryCustomizer<ConfigurableServletWebServerFactory> enableDefaultServlet() {
```
Comment (kwoyke) [inline src/main/java/com/baeldung/spring/MvcConfig.java:113]: Why not public?

File: src/test/java/com/baeldung/test/ChangePasswordIntegrationTest.java [hunk 2]
```diff
@@ -105,8 +99,8 @@ public void givenLoggedInUser_whenChangingPassword_thenCorrect() {
 
         final Response response = request.with().queryParams(params).post(URL);
 
-        assertEquals(200, response.statusCode());
-        assertTrue(response.body().asString().contains("Password updated successfully"));
+        Assertions.assertEquals(200, response.statusCode());
```
Comment (kwoyke) [inline src/test/java/com/baeldung/test/ChangePasswordIntegrationTest.java:102]: Let's use static imports here.

File: src/main/java/com/baeldung/spring/MvcConfig.java (hunk 1 above)
Comment (anuragkumawat) [reply src/main/java/com/baeldung/spring/MvcConfig.java:113]: This bean will not be used outside, Needed this only to enable the default servlet which is required at the start of the application.

File: src/test/java/com/baeldung/test/ChangePasswordIntegrationTest.java (hunk 2 above)
Comment (anuragkumawat) [reply src/test/java/com/baeldung/test/ChangePasswordIntegrationTest.java]: Used the static import for assertions.

**Pull request merged**